src/pkg/services/m365/api/graph/logging.go

```diff
@@ -47,10 +45,11 @@ func logResp(ctx context.Context, resp *http.Response) {
 	// Log api calls according to api debugging configurations.
 	switch respClass {
 	case 2:
+		// only log 2xx's if we want the full response body.
 		if logBody {
 			// only dump the body if it's under a size limit.  We don't want to copy gigs into memory for a log.
 			dump := getRespDump(ctx, resp, os.Getenv(log2xxGraphResponseEnvKey) != "" && resp.ContentLength < logMBLimit)
-			log.Infow("2xx graph api resp", "response", dump)
```

Comment:
Can we please keep this? :) My eyes have grown accustomed to seeing `2xx graph api resp` when i run with `--debug-api-calls`. `2xx` being the very first word makes it easy to catch with the eye.

Reply:
hah, sure, can do